test/accounts.test.js

```diff
@@ -1257,29 +1258,98 @@ describe('vouching cbp earning', async assert => {
   console.log('reset accounts')
   await contracts.accounts.reset({ authorization: `${accounts}@active` })
 
+  console.log('reset settings')
+  await contracts.accounts.reset({ authorization: `${accounts}@active` })
+
+  console.log('configure rewards')
+
+  const cbpRewardResident = 5
+  const cbpRewardCitizen = 7
+
+  await contracts.settings.configure("ref.cbp1.ind", cbpRewardResident, { authorization: `${settings}@active` })
+  await contracts.settings.configure("ref.cbp2.ind", cbpRewardCitizen, { authorization: `${settings}@active` })
+
   console.log('add users')
   await contracts.accounts.adduser(firstuser, 'First user', "individual", { authorization: `${accounts}@active` })
   await contracts.accounts.adduser(seconduser, 'Second user', "individual", { authorization: `${accounts}@active` })
 
-  await contracts.accounts.testsetrs(seconduser, 50, { authorization: `${accounts}@active` })
+  console.log("seconduser is sponsor for first user")
+  await contracts.accounts.addref(seconduser, firstuser, { authorization: `${accounts}@api` })
 
-  console.log('seconduser becomes a resident and can vouch')
-  await contracts.accounts.testresident(seconduser, { authorization: `${accounts}@active` })
+  console.log("firstuser becomes resident")
+  await contracts.accounts.testresident(firstuser, { authorization: `${accounts}@active` })
 
-  console.log('vouch for first user')
-  await contracts.accounts.vouch(seconduser, firstuser, { authorization: `${seconduser}@active` })
+  const cbsAfterResident = await getTableRows({
+    code: accounts,
+    scope: accounts,
+    table: 'cbs',
+    json: true
+  })
 
-  const vouchTable = await getTableRows({
+  // console.log("cbs "+JSON.stringify(cbsAfterResident, null, 2))
+
+  assert({
+    given: 'firstuser became resident',
+    should: 'sponsor received enough cbp',
+    actual: cbsAfterResident.rows[0].community_building_score,
+    expected: cbpRewardResident
+  })
+
+  console.log("firstuser becomes citizen")
+  await contracts.accounts.testcitizen(firstuser, { authorization: `${accounts}@active` })
+  const expected_cbp_cit = await get_settings("ref.cbp2.ind")
+
+  const cbsAfterCitizen = await getTableRows({
     code: accounts,
     scope: accounts,
     table: 'cbs',
     json: true
   })
 
+  // console.log("cbs "+JSON.stringify(cbsAfterCitizen, null, 2))
+
+  assert({
+    given: 'firstuser became citizen',
+    should: 'sponsor received enough cbp',
+    actual: cbsAfterCitizen.rows[0].community_building_score,
+    expected: cbpRewardResident + cbpRewardCitizen
+  })
+
+})
+
+describe.only('Referral cbp reward organization', async assert => {
```

Comment:
I think the `describe.only` is causing that this is the only test being executed

Reply:
Removed